tests/types/types.test.ts

```diff
@@ -1,7 +1,7 @@
 import json from "@eslint/json";
-import { ESLint } from "eslint";
+// import { ESLint } from "eslint";
 
-json satisfies ESLint.Plugin;
+// json satisfies ESLint.Plugin;
```

Comment:
`RuleModule` is `eslint` is not compatible with `RuleDefinition`, so need to comment this out for now. I'll work on updating the types in `eslint` next.